ehr/src/org/labkey/ehr/query/EHRLookupsUserSchema.java

```diff
@@ -271,9 +266,9 @@ private TableInfo createVeterinariansTable(String name, ContainerFilter cf)
                 else
                 {
                     Group assignedGroup = SecurityManager.getGroup(r.getUserId());
-                    if (assignedGroup != null && !assignedGroup.isProjectGroup())
+                    if (assignedGroup != null)
```

Comment:
Not really sure what the original use case was here. I don't see anyone using vets that wouldn't be listed in EHR or having different permissions at project level.

Reply:
I assume I wrote that, but it's been too long to recall. I cannot think of any reason to differentiate between site groups and project-level groups. If no center is actually using a project-level group in this table then it's a moot point and probably doesnt need to exist.

Given how long ago this was, it's equally likely that some long-ago quirk the in SecurityManager API required this kind of check for some reason.